deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/ResourceLifeCycleHandler.java

```diff
@@ -181,9 +188,9 @@ default void waitForResourceDeletion(ApiClient apiClient, List<String> pendingRe
                 String pendingResource = deletePendingResourceIterator.next();
                 try {
                     get(apiClient, pendingResource, namespace);
+                    logger.debug("Wait for deletion, get success");
```

Comment:
Can we make the message more precise like 
"Resource found to be existing, waiting for resource deletion" ?